lib/pages/chat/chat_pinned_events/pinned_events_argument.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:equatable/equatable.dart';
+import 'package:matrix/matrix.dart';
+
+class PinnedEventsArgument with EquatableMixin {
+  final List<Event?> pinnedEvents;
+
+  final Timeline? timeline;
```

Comment:
is it better if we fetch it in Pinned screen?

Reply:
because only way to go to pinned messages screen through the chat screen, and we have loaded it before in chat, so no need to fetch again